Dockerfile

```diff
@@ -72,9 +72,11 @@ RUN git clone https://github.com/OCA/OCB.git -b 8.0 odoo && \
 
 # Execution environment
 USER 0
-ADD sources/odoo.conf /opt/sources/odoo.conf
+ADD sources/odoo.conf /opt/odoo/etc/odoo.conf
```

Comment:
@noahzaozao Actually the odoo.conf should be added by user `odoo`, not `0`. Can you move this line before the `# Execution environment` line please?